src/components/Index/BillContent.svelte

```diff
@@ -15,13 +15,23 @@
 		bills: Pick<Bill, 'id' | 'title' | 'nickname'>[];
 	}
 
+	export let billCategories: string[] = [];
+	export let mpTermChoices: {
+		id: string;
+		value: string;
+		founding_date?: string;
+		dissolution_date?: string;
+	}[] = [];
+
 	const ALL_CATEGORY_KEY = 'ทุกหมวด';
 	const MAX_BILL_BY_STATUS = 3;
 	const MAX_ENACTED_BILL = 10;
 
-	export let billCategories: string[] = [];
+	let selectedCategory = ALL_CATEGORY_KEY;
+
+	let selectedMpTermId = '';
+	$: selectedMpTermId = mpTermChoices[0].id ?? '';
```

Comment:
When use `$` to assign a variable, you don't need to declare it separately. Line 32 can be safely removed.